Delfino/DualCore/SmartWinchDualCore/main.c

```diff
@@ -100,6 +100,17 @@ void init_settings()
 
     modbus_holding_regs[Current_Waypoints_Pointer] = 0;
     modbus_holding_regs[Dwell_Time] = 1;
+    
+    //Defines motor coordinates
+    //5 input lengths required in metres as follows:
+    //Motor A to Motor B
+    //Motor C to Motor D
+    //Motor A to Motor D
+    //Motor A to Motor C
+    //Motor B to Motor D
+    //This fully charactises the layout and coordinates are set to allow for...
+    //any quadrilateral motor shape
+    set_motor_coord(1.0f,1.0f,1.0f,1.4142f,1.4142f);
```

Comment:
why not use parameters taken from modbus_regs?

Reply:
It's in metres as the tension formula require SI units. And I don't understand the second question? That function sets the global motor variable which are stored in modbus_regs, do you want the inputs stored in modbus_regs too?

Reply:
Converting m to mm should be easy as it is a metric system.

Yes, the input should be from modbus too. 
If you predefine the value by hardcoding it, how can the user change the coordinate via modbus?